specification/sections/03-evaluation-context.md

```diff
@@ -52,6 +52,17 @@ API (global) `evaluation context` can be used to supply static data to flag eval
 
 #### Requirement 3.2.2
 
-> Evaluation context **MUST** be merged in the order: API (global) -> client -> invocation, with duplicate values being overwritten.
+> Evaluation context **MUST** be merged in the order: API (global) -> client -> invocation -> before hooks, with duplicate values being overwritten.
```

Comment:
I liked @justinabrahms' language that included (hightest priority) and (lowest priority). It was very clear to me:

>before-hook (highest precedence), invocation, client, api (lowest precedence)

Reply:
```suggestion
> Evaluation context **MUST** be merged in the order: API (global; lowest precedence) -> client -> invocation -> before hooks (highest precedence), with duplicate values being overwritten.
```